### DISCUSSION TITLE
is there support for RN 0.73.x?

### DISCUSSION THREAD

**Discussion by LayMui** [software-mansion/react-native-gesture-handler#2752] [Q&A] [answered]
there is error when using 0.73.x with react-native-gesture-handler
    Invariant Violation: TurboModuleRegistry.getEnforcing(...): 'RNGestureHandlerModule' could not be found. Verify that a module by this name is registered in the native binary.Bridgeless mode: false. TurboModule interop: false. Modules loaded: {"NativeModules":["PlatformConstants","DeviceInfo","KeyboardObserver","I18nManager"],"TurboModules":[],"NotFound":["NativeAnimatedModule","SoundManager","FrameRateLogger","RNGestureHandlerModule"]}

**Comment by m-bert** [comment, marked as answer]
Hi! We are aware of this problem. Most likely it is problem with project setup, not the library itself. This problem is tracked in #2847. I'll close this discussion since you can find more information there.
